src/event-manager.js

```diff
@@ -102,6 +103,17 @@ export default class EventManager {
       });
     }
 
+    // Set recognizer options
+    for (const recognizerName in options.recognizerOptions) {
+      const recognizer = this.manager.get(recognizerName);
+      if (recognizer) {
+        const recognizerOption = options.recognizerOptions[recognizerName];
+        // `enable` is managed by the event registrations
```

Comment:
do you mean `enable` is managed by this class's [`.on(event, handler, srElement)`](https://github.com/uber-web/mjolnir.js/blob/df8a667848495de22d68b250b7a9e4e3e0e56e09/src/event-manager.js#L156)